share/p2p/peers/pool.go

```diff
@@ -23,14 +25,24 @@ type pool struct {
 	cleanupThreshold int
 }
 
+type status int
+
+const (
+	active status = iota
+	cooldown
+	removed
+)
```

Comment:
Don't get confused they are named the same. It is internal pool state model and it could diverge from peer managers sync result feedback API statuscodes

Reply:
I would say if they're functionally the same, lets just use the other defined consts and if they diverge, we can introduce other types later.

Reply:
They just looked similar initially. I've just added another possible result value for Manager so status model for manager and pool doesn't look same anymore. Also naming of statuses is more explicit now so it is less confusing what DoneFunc call would do internally.